modules/monitoring-alertmanager-default-configuration.adoc

```diff
@@ -28,4 +28,6 @@ receivers:
 
 {product-title} monitoring ships with the Watchdog alert that always triggers by default to ensure the availability of the monitoring infrastructure.
 
-// FIXME perhaps explain what individual entries in the listing mean
+Watchdog is a simple Prometheus alerting rule that always triggers. The Alertmanager continuously sends notifications for Watchdog to the notification provider that supports this functionality. The notification provider can be configured to issue alerts to the administrator if the monitoring stack fails. An example of a third-party notification provider is PagerDuty.
```

Comment:
"is always active" rather than "always triggers"?

"The Alertmanager continuously sends notifications for Watchdog to the notification provider that supports this functionality. The notification provider can be configured to issue alerts to the administrator if the monitoring stack fails." -> "Alertmanager continuously sends notifications for the Watchdog alert to the notification provider which is configured to notify the administrator when it stops receiving the Watchdog alert."

Reply:
Thank you, @simonpasquier, good feedback.
* "is always active" might imply that it is enabled, but not firing. How about "fires continuously"?
* Your suggested sentences assume that the provider has been configured to alert the administrator. While it's reasonable, I don't think we can assume that, so let's go with "usually".

Could you take another look at the resultant changes? Thank you.